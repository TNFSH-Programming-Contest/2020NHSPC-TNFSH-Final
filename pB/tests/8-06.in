882
-463.359 829.638
-457.437 832.918
-451.492 836.156
-445.524 839.351
-439.533 842.503
-433.521 845.613
-427.486 848.68
-421.429 851.704
-415.351 854.684
-409.252 857.621
-403.132 860.515
-396.992 863.365
-390.831 866.171
-384.651 868.933
-378.451 871.651
-372.232 874.325
-365.994 876.955
-359.738 879.54
-353.463 882.08
-347.17 884.576
-340.86 887.026
-334.533 889.432
-328.188 891.793
-321.827 894.108
-315.449 896.378
-309.056 898.602
-302.646 900.781
-296.222 902.914
-289.782 905.002
-283.328 907.043
-276.859 909.038
-270.376 910.987
-263.88 912.89
-257.37 914.747
-250.847 916.557
-244.311 918.321
-237.763 920.038
-231.203 921.709
-224.631 923.332
-218.048 924.909
-211.454 926.439
-204.849 927.922
-198.233 929.357
-191.608 930.746
-184.972 932.087
-178.328 933.381
-171.674 934.628
-165.012 935.827
-158.341 936.979
-151.662 938.083
-144.976 939.14
-138.282 940.149
-131.581 941.11
-124.873 942.023
-118.159 942.889
-111.439 943.707
-104.714 944.477
-97.983 945.199
-91.247 945.873
-84.507 946.499
-77.762 947.077
-71.013 947.607
-64.261 948.089
-57.505 948.522
-50.747 948.908
-43.986 949.245
-37.223 949.535
-30.457 949.776
-23.691 949.969
-16.923 950.113
-10.154 950.21
-3.385 950.258
3.385 950.258
10.154 950.21
16.923 950.113
23.691 949.969
30.457 949.776
37.223 949.535
43.986 949.245
50.747 948.908
57.505 948.522
64.261 948.089
71.013 947.607
77.762 947.077
84.507 946.499
91.247 945.873
97.983 945.199
104.714 944.477
111.439 943.707
118.159 942.889
124.873 942.023
131.581 941.11
138.282 940.149
144.976 939.14
151.662 938.083
158.341 936.979
165.012 935.827
171.674 934.628
178.328 933.381
184.972 932.087
191.608 930.746
198.233 929.357
204.849 927.922
211.454 926.439
218.048 924.909
224.631 923.332
231.203 921.709
237.763 920.038
244.311 918.321
250.847 916.557
257.37 914.747
263.88 912.89
270.376 910.987
276.859 909.038
283.328 907.043
289.782 905.002
296.222 902.914
302.646 900.781
309.056 898.602
315.449 896.378
321.827 894.108
328.188 891.793
334.533 889.432
340.86 887.026
347.17 884.576
353.463 882.08
359.738 879.54
365.994 876.955
372.232 874.325
378.451 871.651
384.651 868.933
390.831 866.171
396.992 863.365
403.132 860.515
409.252 857.621
415.351 854.684
421.429 851.704
427.486 848.68
433.521 845.613
439.533 842.503
445.524 839.351
451.492 836.156
457.437 832.918
463.359 829.638
469.257 826.317
475.132 822.953
480.982 819.547
486.808 816.1
492.61 812.611
498.386 809.081
504.137 805.511
509.863 801.899
515.562 798.246
521.236 794.553
526.883 790.82
532.503 787.047
538.096 783.233
543.662 779.38
549.2 775.487
554.711 771.555
560.193 767.584
565.647 763.574
571.072 759.525
576.468 755.438
581.835 751.312
587.172 747.148
592.48 742.946
597.757 738.707
603.005 734.43
608.221 730.115
613.407 725.764
618.561 721.376
623.685 716.951
628.776 712.49
633.836 707.993
638.863 703.459
643.858 698.891
648.821 694.286
653.75 689.646
658.646 684.972
663.509 680.262
668.338 675.519
673.134 670.74
677.895 665.928
682.621 661.082
687.314 656.202
691.971 651.29
696.593 646.344
701.179 641.365
705.731 636.354
710.246 631.31
714.725 626.234
719.168 621.127
723.575 615.988
727.944 610.818
732.277 605.617
736.573 600.385
740.831 595.122
745.052 589.83
749.235 584.507
753.38 579.155
757.486 573.774
761.554 568.363
765.584 562.923
769.575 557.455
773.526 551.959
777.439 546.434
781.312 540.882
785.145 535.303
788.938 529.696
792.692 524.062
796.405 518.402
800.078 512.716
803.71 507.003
807.301 501.265
810.852 495.501
814.361 489.712
817.829 483.898
821.255 478.06
824.64 472.198
827.983 466.311
831.284 460.401
834.542 454.468
837.759 448.511
840.932 442.532
844.064 436.53
847.152 430.506
850.197 424.46
853.199 418.393
856.158 412.304
859.074 406.195
861.945 400.064
864.774 393.914
867.558 387.744
870.298 381.554
872.994 375.344
875.646 369.116
878.253 362.868
880.816 356.603
883.334 350.319
885.807 344.017
888.235 337.698
890.618 331.362
892.956 325.009
895.249 318.64
897.496 312.254
899.697 305.853
901.853 299.436
903.964 293.004
906.028 286.557
908.046 280.095
910.019 273.619
911.945 267.13
913.825 260.627
915.658 254.11
917.445 247.581
919.185 241.039
920.879 234.485
922.526 227.919
924.126 221.341
925.68 214.752
927.186 208.152
928.645 201.542
930.058 194.922
931.423 188.291
932.74 181.651
934.011 175.002
935.234 168.344
936.409 161.677
937.537 155.002
938.618 148.32
939.65 141.629
940.635 134.932
941.573 128.228
942.462 121.517
943.304 114.8
944.098 108.077
944.844 101.349
945.542 94.616
946.192 87.878
946.794 81.135
947.348 74.388
947.854 67.638
948.312 60.884
948.721 54.127
949.083 47.367
949.396 40.605
949.661 33.84
949.878 27.074
950.047 20.307
950.167 13.539
950.24 6.769
950.264 0.0
950.24 -6.769
950.167 -13.539
950.047 -20.307
949.878 -27.074
949.661 -33.84
949.396 -40.605
949.083 -47.367
948.721 -54.127
948.312 -60.884
947.854 -67.638
947.348 -74.388
946.794 -81.135
946.192 -87.878
945.542 -94.616
944.844 -101.349
944.098 -108.077
943.304 -114.8
942.462 -121.517
941.573 -128.228
940.635 -134.932
939.65 -141.629
938.618 -148.32
937.537 -155.002
936.409 -161.677
935.234 -168.344
934.011 -175.002
932.74 -181.651
931.423 -188.291
930.058 -194.922
928.645 -201.542
927.186 -208.152
925.68 -214.752
924.126 -221.341
922.526 -227.919
920.879 -234.485
919.185 -241.039
917.445 -247.581
915.658 -254.11
913.825 -260.627
911.945 -267.13
910.019 -273.619
908.046 -280.095
906.028 -286.557
903.964 -293.004
901.853 -299.436
899.697 -305.853
897.496 -312.254
895.249 -318.64
892.956 -325.009
890.618 -331.362
888.235 -337.698
885.807 -344.017
883.334 -350.319
880.816 -356.603
878.253 -362.868
875.646 -369.116
872.994 -375.344
870.298 -381.554
867.558 -387.744
864.774 -393.914
861.945 -400.064
859.074 -406.195
856.158 -412.304
853.199 -418.393
850.197 -424.46
847.152 -430.506
844.064 -436.53
840.932 -442.532
837.759 -448.511
834.542 -454.468
831.284 -460.401
827.983 -466.311
824.64 -472.198
821.255 -478.06
817.829 -483.898
814.361 -489.712
810.852 -495.501
807.301 -501.265
803.71 -507.003
800.078 -512.716
796.405 -518.402
792.692 -524.062
788.938 -529.696
785.145 -535.303
781.312 -540.882
777.439 -546.434
773.526 -551.959
769.575 -557.455
765.584 -562.923
761.554 -568.363
757.486 -573.774
753.38 -579.155
749.235 -584.507
745.052 -589.83
740.831 -595.122
736.573 -600.385
732.277 -605.617
727.944 -610.818
723.575 -615.988
719.168 -621.127
714.725 -626.234
710.246 -631.31
705.731 -636.354
701.179 -641.365
696.593 -646.344
691.971 -651.29
687.314 -656.202
682.621 -661.082
677.895 -665.928
673.134 -670.74
668.338 -675.519
663.509 -680.262
658.646 -684.972
653.75 -689.646
648.821 -694.286
643.858 -698.891
638.863 -703.459
633.836 -707.993
628.776 -712.49
623.685 -716.951
618.561 -721.376
613.407 -725.764
608.221 -730.115
603.005 -734.43
597.757 -738.707
592.48 -742.946
587.172 -747.148
581.835 -751.312
576.468 -755.438
571.072 -759.525
565.647 -763.574
560.193 -767.584
554.711 -771.555
549.2 -775.487
543.662 -779.38
538.096 -783.233
532.503 -787.047
526.883 -790.82
521.236 -794.553
515.562 -798.246
509.863 -801.899
504.137 -805.511
498.386 -809.081
492.61 -812.611
486.808 -816.1
480.982 -819.547
475.132 -822.953
469.257 -826.317
463.359 -829.638
457.437 -832.918
451.492 -836.156
445.524 -839.351
439.533 -842.503
433.521 -845.613
427.486 -848.68
421.429 -851.704
415.351 -854.684
409.252 -857.621
403.132 -860.515
396.992 -863.365
390.831 -866.171
384.651 -868.933
378.451 -871.651
372.232 -874.325
365.994 -876.955
359.738 -879.54
353.463 -882.08
347.17 -884.576
340.86 -887.026
334.533 -889.432
328.188 -891.793
321.827 -894.108
315.449 -896.378
309.056 -898.602
302.646 -900.781
296.222 -902.914
289.782 -905.002
283.328 -907.043
276.859 -909.038
270.376 -910.987
263.88 -912.89
257.37 -914.747
250.847 -916.557
244.311 -918.321
237.763 -920.038
231.203 -921.709
224.631 -923.332
218.048 -924.909
211.454 -926.439
204.849 -927.922
198.233 -929.357
191.608 -930.746
184.972 -932.087
178.328 -933.381
171.674 -934.628
165.012 -935.827
158.341 -936.979
151.662 -938.083
144.976 -939.14
138.282 -940.149
131.581 -941.11
124.873 -942.023
118.159 -942.889
111.439 -943.707
104.714 -944.477
97.983 -945.199
91.247 -945.873
84.507 -946.499
77.762 -947.077
71.013 -947.607
64.261 -948.089
57.505 -948.522
50.747 -948.908
43.986 -949.245
37.223 -949.535
30.457 -949.776
23.691 -949.969
16.923 -950.113
10.154 -950.21
3.385 -950.258
-3.385 -950.258
-10.154 -950.21
-16.923 -950.113
-23.691 -949.969
-30.457 -949.776
-37.223 -949.535
-43.986 -949.245
-50.747 -948.908
-57.505 -948.522
-64.261 -948.089
-71.013 -947.607
-77.762 -947.077
-84.507 -946.499
-91.247 -945.873
-97.983 -945.199
-104.714 -944.477
-111.439 -943.707
-118.159 -942.889
-124.873 -942.023
-131.581 -941.11
-138.282 -940.149
-144.976 -939.14
-151.662 -938.083
-158.341 -936.979
-165.012 -935.827
-171.674 -934.628
-178.328 -933.381
-184.972 -932.087
-191.608 -930.746
-198.233 -929.357
-204.849 -927.922
-211.454 -926.439
-218.048 -924.909
-224.631 -923.332
-231.203 -921.709
-237.763 -920.038
-244.311 -918.321
-250.847 -916.557
-257.37 -914.747
-263.88 -912.89
-270.376 -910.987
-276.859 -909.038
-283.328 -907.043
-289.782 -905.002
-296.222 -902.914
-302.646 -900.781
-309.056 -898.602
-315.449 -896.378
-321.827 -894.108
-328.188 -891.793
-334.533 -889.432
-340.86 -887.026
-347.17 -884.576
-353.463 -882.08
-359.738 -879.54
-365.994 -876.955
-372.232 -874.325
-378.451 -871.651
-384.651 -868.933
-390.831 -866.171
-396.992 -863.365
-403.132 -860.515
-409.252 -857.621
-415.351 -854.684
-421.429 -851.704
-427.486 -848.68
-433.521 -845.613
-439.533 -842.503
-445.524 -839.351
-451.492 -836.156
-457.437 -832.918
-463.359 -829.638
-469.257 -826.317
-475.132 -822.953
-480.982 -819.547
-486.808 -816.1
-492.61 -812.611
-498.386 -809.081
-504.137 -805.511
-509.863 -801.899
-515.562 -798.246
-521.236 -794.553
-526.883 -790.82
-532.503 -787.047
-538.096 -783.233
-543.662 -779.38
-549.2 -775.487
-554.711 -771.555
-560.193 -767.584
-565.647 -763.574
-571.072 -759.525
-576.468 -755.438
-581.835 -751.312
-587.172 -747.148
-592.48 -742.946
-597.757 -738.707
-603.005 -734.43
-608.221 -730.115
-613.407 -725.764
-618.561 -721.376
-623.685 -716.951
-628.776 -712.49
-633.836 -707.993
-638.863 -703.459
-643.858 -698.891
-648.821 -694.286
-653.75 -689.646
-658.646 -684.972
-663.509 -680.262
-668.338 -675.519
-673.134 -670.74
-677.895 -665.928
-682.621 -661.082
-687.314 -656.202
-691.971 -651.29
-696.593 -646.344
-701.179 -641.365
-705.731 -636.354
-710.246 -631.31
-714.725 -626.234
-719.168 -621.127
-723.575 -615.988
-727.944 -610.818
-732.277 -605.617
-736.573 -600.385
-740.831 -595.122
-745.052 -589.83
-749.235 -584.507
-753.38 -579.155
-757.486 -573.774
-761.554 -568.363
-765.584 -562.923
-769.575 -557.455
-773.526 -551.959
-777.439 -546.434
-781.312 -540.882
-785.145 -535.303
-788.938 -529.696
-792.692 -524.062
-796.405 -518.402
-800.078 -512.716
-803.71 -507.003
-807.301 -501.265
-810.852 -495.501
-814.361 -489.712
-817.829 -483.898
-821.255 -478.06
-824.64 -472.198
-827.983 -466.311
-831.284 -460.401
-834.542 -454.468
-837.759 -448.511
-840.932 -442.532
-844.064 -436.53
-847.152 -430.506
-850.197 -424.46
-853.199 -418.393
-856.158 -412.304
-859.074 -406.195
-861.945 -400.064
-864.774 -393.914
-867.558 -387.744
-870.298 -381.554
-872.994 -375.344
-875.646 -369.116
-878.253 -362.868
-880.816 -356.603
-883.334 -350.319
-885.807 -344.017
-888.235 -337.698
-890.618 -331.362
-892.956 -325.009
-895.249 -318.64
-897.496 -312.254
-899.697 -305.853
-901.853 -299.436
-903.964 -293.004
-906.028 -286.557
-908.046 -280.095
-910.019 -273.619
-911.945 -267.13
-913.825 -260.627
-915.658 -254.11
-917.445 -247.581
-919.185 -241.039
-920.879 -234.485
-922.526 -227.919
-924.126 -221.341
-925.68 -214.752
-927.186 -208.152
-928.645 -201.542
-930.058 -194.922
-931.423 -188.291
-932.74 -181.651
-934.011 -175.002
-935.234 -168.344
-936.409 -161.677
-937.537 -155.002
-938.618 -148.32
-939.65 -141.629
-940.635 -134.932
-941.573 -128.228
-942.462 -121.517
-943.304 -114.8
-944.098 -108.077
-944.844 -101.349
-945.542 -94.616
-946.192 -87.878
-946.794 -81.135
-947.348 -74.388
-947.854 -67.638
-948.312 -60.884
-948.721 -54.127
-949.083 -47.367
-949.396 -40.605
-949.661 -33.84
-949.878 -27.074
-950.047 -20.307
-950.167 -13.539
-950.24 -6.769
-950.264 0.0
-950.24 6.769
-950.167 13.539
-950.047 20.307
-949.878 27.074
-949.661 33.84
-949.396 40.605
-949.083 47.367
-948.721 54.127
-948.312 60.884
-947.854 67.638
-947.348 74.388
-946.794 81.135
-946.192 87.878
-945.542 94.616
-944.844 101.349
-944.098 108.077
-943.304 114.8
-942.462 121.517
-941.573 128.228
-940.635 134.932
-939.65 141.629
-938.618 148.32
-937.537 155.002
-936.409 161.677
-935.234 168.344
-934.011 175.002
-932.74 181.651
-931.423 188.291
-930.058 194.922
-928.645 201.542
-927.186 208.152
-925.68 214.752
-924.126 221.341
-922.526 227.919
-920.879 234.485
-919.185 241.039
-917.445 247.581
-915.658 254.11
-913.825 260.627
-911.945 267.13
-910.019 273.619
-908.046 280.095
-906.028 286.557
-903.964 293.004
-901.853 299.436
-899.697 305.853
-897.496 312.254
-895.249 318.64
-892.956 325.009
-890.618 331.362
-888.235 337.698
-885.807 344.017
-883.334 350.319
-880.816 356.603
-878.253 362.868
-875.646 369.116
-872.994 375.344
-870.298 381.554
-867.558 387.744
-864.774 393.914
-861.945 400.064
-859.074 406.195
-856.158 412.304
-853.199 418.393
-850.197 424.46
-847.152 430.506
-844.064 436.53
-840.932 442.532
-837.759 448.511
-834.542 454.468
-831.284 460.401
-827.983 466.311
-824.64 472.198
-821.255 478.06
-817.829 483.898
-814.361 489.712
-810.852 495.501
-807.301 501.265
-803.71 507.003
-800.078 512.716
-796.405 518.402
-792.692 524.062
-788.938 529.696
-785.145 535.303
-781.312 540.882
-777.439 546.434
-773.526 551.959
-769.575 557.455
-765.584 562.923
-761.554 568.363
-757.486 573.774
-753.38 579.155
-749.235 584.507
-745.052 589.83
-740.831 595.122
-736.573 600.385
-732.277 605.617
-727.944 610.818
-723.575 615.988
-719.168 621.127
-714.725 626.234
-710.246 631.31
-705.731 636.354
-701.179 641.365
-696.593 646.344
-691.971 651.29
-687.314 656.202
-682.621 661.082
-677.895 665.928
-673.134 670.74
-668.338 675.519
-663.509 680.262
-658.646 684.972
-653.75 689.646
-648.821 694.286
-643.858 698.891
-638.863 703.459
-633.836 707.993
-628.776 712.49
-623.685 716.951
-618.561 721.376
-613.407 725.764
-608.221 730.115
-603.005 734.43
-597.757 738.707
-592.48 742.946
-587.172 747.148
-581.835 751.312
-576.468 755.438
-571.072 759.525
-565.647 763.574
-560.193 767.584
-554.711 771.555
-549.2 775.487
-543.662 779.38
-538.096 783.233
-532.503 787.047
-526.883 790.82
-521.236 794.553
-515.562 798.246
-509.863 801.899
-504.137 805.511
-498.386 809.081
-492.61 812.611
-486.808 816.1
-480.982 819.547
-475.132 822.953
-469.257 826.317
-945.97 212.369
