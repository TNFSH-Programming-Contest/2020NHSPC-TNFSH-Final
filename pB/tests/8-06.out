11577336.583
